packages/next/server/dev/next-dev-server.ts

```diff
@@ -693,7 +693,7 @@ export default class DevServer extends Server {
   ) {
     let usedOriginalStack = false
 
-    if (isError(err) && err.name && err.stack && err.message) {
+    if (isError(err) && err.stack) {
```

Comment:
isError already checks if `name` and `message` are there, is the second truthiness check necessary?

Reply:
Looks good to drop the redundant checks here 👍